runner/options.go

```diff
@@ -396,6 +398,7 @@ func ParseOptions() *Options {
 		flagSet.BoolVarP(&options.ExcludeCDN, "exclude-cdn", "ec", false, "skip full port scans for CDNs (only checks for 80,443)"),
 		flagSet.IntVar(&options.Retries, "retries", 0, "number of retries"),
 		flagSet.IntVar(&options.Timeout, "timeout", 5, "timeout in seconds"),
+		flagSet.StringVar(&options.Delay, "delay", "", "duration between each http request (eg: 200ms, 1s)"),
```

Comment:
I think we can simplify with `flagSet.DurationVarP`

Reply:
Updated code as per your suggestion.